plugins/talk-plugin-auth/client/components/SignInContainer.js

```diff
@@ -60,6 +60,7 @@ class SignInContainer extends React.Component {
     if (e.key === 'auth') {
       const { err, data } = JSON.parse(e.newValue);
       authCallback(err, data);
+      localStorage.removeItem('auth');
```

Comment:
Because there is still a listener to the `storage` when this function mutates the storage, won't this cause a loop until the component is unmounted?

Can you check the `newValue` property of `e` to see if the value is a string? (See https://developer.mozilla.org/en-US/docs/Web/Events/storage)